---
title: 'Sex-Specific Evolution of the Meiotic Recombination Rate'
date: '`r Sys.Date()`'
author: "April L. Peterson, Bret Payseur"
output:
  html_document:
  word_document:
  pdf_document:
    includes:
      in_header: header.tex
always_allow_html: yes
bibliography: refs_Main_Report.bib
csl: elife.csl
---



```{r set.up, echo=FALSE, warning=FALSE, message=FALSE}

#library(tidyverse)
library(knitr)
library(ggplot2)
#library(pwr)

library(dplyr)
library(plyr)
library(lattice)
library(raster)
library(lme4)
library(nlme)
library(RLRsim)
library(ggpubr)
#library(reshape2)
#library(patchwork)
library(png)
library(grid)
library(gridBase)
library(gridExtra)
library(cowplot)
library(kableExtra)
library(xlsx)
library(reshape2)

options(digits=2)

#setwd("~.")#switch over to here()

load(file="~./MLH1repo/data/Results_setup_5.30.20.RData")
#load(file="~./MLH1repo/data/Results_setup_5.21.20.RData") #Results_setup_5.14.20.RData

#load(file="~./MLH1repo/data/CleanBivData_6.3.20.RData")

#
#load(file="~./MLH1repo/data/total_SC_30.12.19.RData")

#skeletonize - this is the total sc file
#SC.skel = read.csv("~./MLH1repo/data/SCskel_output_feb20.csv", header = TRUE, strip.white = TRUE)


#load functions
source("~./MLH1repo/src/CommonFunc_MLH1repo.R")
```

# ABSTRACT

Although meiotic recombination is required for successful gametogenesis in most species that reproduce sexually, the rate of crossing over varies among individuals. **define genome-wide recombination rate**.

Differences in recombination rate between females and males are perhaps the most striking form of this variation. Existing data fail to directly address the extent to which recombination experiences similar evolutionary pressures. To fill this gap, we measured meiotic recombination in both sexes for a panel of house mouse cross three subspecies. Using inbred strains and single cell immunohistochemistry allowed us to place sex-specific observations within the same genetic background and meiotic context. Our results indicate highly discordant evolutionary patterns in the two sexes. Whereas male recombination rates show evidence of rapid evolution over short evolutionary timescales, female recombination rates measured in the same strains are mostly static. These results strongly indicate that house mouse has two genome wide recombination rates which display distinct evolutionary trajectories. 


# INTRODUCTION

### 0.   Importance of recombination rate

Meiosis can be reduced to the expression of (2n -> 4n -> 2n -> 1n) which tracks the duplication of a diploid genome into haploid cell products. The meiotic program relies on crossovers and the process of recombination to ensure the correct separation of chromosomes. The total number of COs per cell (at the 4n stage) is equal to the genome wide recombination rate (gwRR). **The genome wide recombination rate is a cell based metric for genome wide recombination rate is tightly connected to organisms fitness through correct chromosome segregation and fertility.**

**at the fine scale / in regards to genetic variation**, In the indriect manner, the recombination rate regulates populations’ response to selection, and determine the fate of novel mutations by transferring beneficial mutations onto novel genetic backgrounds or by breaking linkage of negative mutations from beneficial genetic backgrounds (cite).  This process shapes the genomic patterns of genetic variation, with high recombination areas of genome having more nucleotide variation while areas of low recombination have decreased genetic variation (Begun Aquadro, Nachman, Payseur).

**but back to the genome-wide recombination rate**, The process of chromosome segregation is integral to the process of producing haploid cells and can be considered a constraint in regards to the evolution. When the genome wide recombination rate is framed within the single cell context it is closely connected to the  contraints which are imposed at the single cell level. Reflecting the importance of it's role in chromosome segreation is that the  number of haploid pairs is strong predictor for genome wide RR across large species(taxa) / divergence scales [@ottoPaysuer2019; @stapley_variation_2017]. (the number of crossovers per chromosome are highly conserved)<However, whie the rate of recombination is highly variable, the count of crossovers per chromosome is conserved between 1 to 3 (cite), suggesting that such thresholds/constraints may be behind the overall patterns.>

**connection of single cell based gwRR to thresholds imposed at the single cell level** (why is gwRR the better scale for evolutionary patterns?)


### 1. Understanding levels of variation in gwRR

Because we know the meiotic programs, we have expectations for where the thresholds(constraints) of the gwRR 
the rate will fall. First, the lower threshold is defined as an obligate crossover per homolog pair which promotes proper chromosome segregation [@nagaoka2012]. second (while there is less evidence,), the upper threshold maybe regulated/shaped/affected due to rates of ectopic recombination or the programmed DNA damage preceding crossovers act as constraints to the upper threshold of meiotic recombination (cite). Yet, Within these thresholds the genome wide recombination rate still varies across species and indiviuals. 
which is the subject of an open question in the field.  **the gwRR is the best metric for since both of these constraints are thought to act at the cell wide level.**

**Sex is one of the most notable axes along which individuals vary**. Sexual dimorphism in recombination rates is called heterochiasmy [@lenormand2003]. An understanding of how sex shapes the evolution of recombination cannot be achieved with available data. Comprehensive comparisons of female and male recombination rates usually come from outbred populations humans [@Kong2004;, @Kong2008; @Kong2014; @halldorsson2019], dog,  cattle [@ma2015_cattle; @Shen2018_cattle], (other sheep) sheep [@johnston2016_soay], and mouse (CC cite) in which the role of sex is confounded with the contributions of genetic variation. Under anisogamy where a species has distinct gametes, there is no first principle which would predict the evolution of sexually dimorphic recombination rates. Yet heterochiasmy, is commonly observed in dioecious species, suggesting that other meiotic traits which distinguish the gametes and their meiotic programs, for example symmetrical vs asymmetrical cell division, may impose selection for sexually dimorphic recombination rates. Although it is clear that the relationship between female and male recombination rates can differ among species, comparisons between and within closely related species are missing. Direct contrasts between females and males across a common, diverse set of genomic backgrounds would reveal whether the recombination rate evolves differently in the sexes.

<It’s important to note, – that the genome wide recombination rate can be decomposed – and connected to related traits (which also show conserved patterns of sexual dimorphism>

**understanding heterochiasmy and sexual dimorphism in meiosis have advanced since first being documented (morgan 1914) -- more than just the direction -- more evidence is accumulating that there are conserved patterns of sexual dimorphism in meiotic recombinatio - related to recombination rate**. Recently two specific features have been highlighted as conserved patterns of heterochiasmy:

@CahoonLibuda2019 show In species with meiotic chromatin (SC axis length) quantified for both sexes, longer chromosome axis generally has higher recombination rates. 
@sardell_sex_2020, A survey of 51 species shows there are conserved broad scale recombination landscape differences. Generally males have telomere-bias crossover placement and females have more uniform placement.

These observations raise the question of conservation in sexually patterns are maintained with the gwRR and it’s decomposed traits


**remember these points kinda distract from gwRR aspect**


### 2.  The House Mouse is a great model for uncovering evolutionary patterns at a short timescale

The House mouse complex arose from a recent radiation providing an opportunity to interrogate natural variation at short evolutionary scales, potentially narrowing the mutational space for identifying variants which would cause gwRR variation. Add divergence ranges. divergence spanning 0.5 mya to 5 mya.

Classical lab strains of mice have generated extensive knowledge central to meiosis and outcomes on recombination through extensive studies on the genes involved in the meiotic recombination pathway (cite) and previous crosses for understanding the genetic architecture of recombination rate variation @dumont2011house; @murdoch2010; @Wang2017island].

**Wild derived inbred strains generate the best comparison of females and males, besides the sex chromosomes, the mouse for each genome are almost identical**. Inbred strains enable one of the most direct comparison of both female and male --  versions of meiosis --enable use to conclude that sex is a primary factor -- 

Additionally due to their global distribution allows us to compare samples of natural diversity from a broad geographic range.  (wild derived inbred strains – will capture genetic diversity not represented in classical lab strains.)

While there are strains with Robertsonian translocations, in house mouse and related murid species it is possible to assemble sets of strains with identical karyotypes, 20 pairs of acrocentric chromosomes.

House mouse is well suited for single cell cytology approaches. The single cell level allows integration of data at a closer connection to the molecular pathway and meiotic program. In addition to the quantification of gwRR. This study quantifies precursors to crossovers, double strand breaks (DSBs), and meiotic chromosome morphology across stages of meiosis which would not be accessible to with genetic linkage data and crosses.


### 3. What we accomplished in this paper

  For the first time in many of these strains, We report a rare, direct, evolutionary comparison of recombination rate in females and males for this short evolutionary scale. We use rare strains from a broad range of geographic locations of the species territory.

The gwRR is made up of crossovers occurring on individual chromosomes within cells. Thus we quantified meiotic chromosome morphology (SC length) and placement of crossovers to comprise an approximate picture of the recombination landscape.

Our results indicate rapid male specific evolution of gwRR occurred in a subset of house mouse lines. We observed up to a 30% difference (translating into approximately ~7 more crossovers per cell), a surprising amount considering the short evolutionary time scale.



# MATERIALS AND METHODS


### Mice

<new>
We used a panel of wild-derived inbred strains of house mice (*Mus musculus*) and related murid species to profile natural genetic variation in recombination (Table 1). Our survey included 5 strains from *M. m. musculus*, 4 strains from *M. m. domesticus*, 2 strains from *M. m. molossinus*, 2 strains from *M. m. castaneus*, and 1 strain each from *M. spicilegus* and *M. spretus*. <removed _M. caroli_> We subsequently denote strains by their abbreviated subspecies and name (e.g. *domesticus^WSB^*). Mice were housed at dedicated, temperature-controlled facilities in the UW-Madison School of Medicine and Public Health, with the exception of mice from Gough Island, which were housed in a temperature-controlled facility in the UW-Madison School of Veterinary Medicine. Mice from an inbred strain of Gough Island mice were sampled after **XX** <track down the generation of Gough inbred levels> generations of brother-sister mating. All mice were provided with ad libitum food and water. Procedures followed protocols approved by IACUC.


### Tissue Collection and Immunohistochemistry

<new>
The same dry-down spread technique was applied to both spermatocytes and oocytes, following [@peters_1997], with adjustment for volumes. Spermatocyte spreads were collected and prepared as described in [@peterson2019]. The majority of mice used for MLH1 counts were between 5 and 12 weeks of age. Juvenile mice between 12 and 15 days of age were used for DMC1 counts. Both ovaries were collected from embryos (16-21 embryonic days) or neonates (0-48 hours after birth). Whole testes were incubated in 3ml of hypotonic solution for 45 minutes. Decapsulated ovaries were incubated in 300ul of hypotonic solution for 45 minutes. Fifteen microliters of cell slurry (masticated gonads) were transferred to 80ul of 2% PFA solution. Cells were fixed in this solution and dried in a humid chamber at room temperature overnight. The following morning, slides were treated with a Photoflow wash (Kodak, cite). Slides were stored at -20\*C if not stained immediately. To visualize the structure of meiotic chromosomes, we used antibody markers for the centromere (CREST) and lateral element of the synaptonemal complex (SC) (SYCP3). Crossovers (COs) were visualized as MLH1 foci. Double strand breaks (DSBs) were visualized as DMC1 foci. The staining protocol followed [@anderson1999] and [@koehler2002]. Antibody staining and slide blocking were performed in 1X antibody dilution buffer (ADB) (normal donkey serum (Jackson ImmunoResearch), 1X PBS, bovine serum albumin (Sigma), and Triton X-100 (Sigma)). Following a 30-minute blocking wash in ABD, each slide was incubated with 60ul of a primary antibody master mix for 48 hours at 37\*C. The master mix recipe contained polyclonal anti-rabbit anti-MLH1 (Calbiochem; diluted 1:50) or anti-rabbit anti-DMC1 (mix of DMC1), anti-goat polyclonal anti-SYCP3, (Abcam; diluted 1:50), and anti-human polyclonal antibody to CREST (Antibodies, Inc; diluted 1:200) suspended in ADB. Slides were washed twice in 50ml ADB before the first round of secondary antibody incubation for 12 hours at 37\*C. Alexa Fluor 488 donkey anti-rabbit IgG (Invitrgoen, location; diluted to 1:100) and Coumarin AMCA donkey anti-human IgG (Jackson ImmunoResearch; diluted to 1:200) were suspended in ADB. The last incubation of Alexa Fluor 568 donkey anti-goat (Invitrogen; diluted 1:100) was incubated at 1:100 for 2 hours at 37\* C. Slides were fixed with Prolong Gold Antifade (Invitrogen) for 24 hours after a final wash in 1x PBS.

### Image Processing

<new>
Images were captured using a Zeiss Axioplan 2 microscope with AxioLab camera and AxioVision software (Zeiss, Cambridge, UK). Preprocessing, including cropping, noise reduction, and histogram adjustments, was performed using Photoshop (v13.0). Image file names were anonymized before manual scoring of MLH1 foci or DMC1 foci using Photoshop.


### Analysis

<new 1/n>

To estimate the number of crossovers across the genome, we counted MLH1 foci. MLH1 foci were counted in cells with intact and complete karyotypes (19 acrocentric bivalents and XY for spermatocytes; 20 acrocentric bivalents for oocytes) and distinct MLH1 foci. A quality score ranging from 1 (best) to 5 (worst) was assigned to each cell based on visual appearance of staining and spread of bivalents. Cells with a score of 5 were excluded from the final analysis. Distributions of MLH1 count per cell were visually inspected for normality (Supplemental Figure 1). MLH1 foci located on the XY in spermatocytes were excluded from counts. In addition to MLH1 counts, we measured several traits to further characterize the recombination landscape. To estimate the number of double-strand breaks, a minority of which lead to crossovers, mean DMC1 foci per cell was quantified for a single male from each of a subset of strains (*molossinus^MSM^*, _musculus^PWD^_,  _domesticus^WSB^_, and _domesticus^G^_ ). SC morphology and CREST foci number were used to stage spermatocytes as early zygotene or late zygotene.

<new 2/n>
To measure bivalent SC length, two image analysis algorithms were used. The first algorithm estimates the total (summed) SC length across bivalents for individual cells (@wang2019_SC). The second algorithm estimates the SC length of individual bivalents (@peterson2019). Both algorithms apply a ‘skeletonizing’ transformation to synapsed chromosomes that produces a single, pixel-wide ‘trace’ of the bivalent shape. Total SC length per cell was quantified from pachytene cell images (@wang2019_SC). 

To reduce algorithmic errors in SC isolation, outliers were visually identified at the mouse level and removed from the data set. Mouse averages were calculated from cell-wide total SC lengths in `r formatC( length(  (Total.SC_DF  %>% filter(species == "M.musculus")  )$Random.Name), format="d", big.mark=",")` out of `r formatC( length(  (MLH1_data %>% filter(species == "M.musculus") )$Random.Name), format="d", big.mark=",")` cells with MLH1 counts. SC length of individual bivalents was quantified in pachytene cell images (@peterson2019). The DNA CrossOver algorithm (@peterson2019) isolates single, straightened bivalent shapes, returning SC length, location of MLH1 foci, and location of CREST (centromere) foci. The algorithm substantially speeds the accurate measurement of bivalents, but it sometimes interprets overlapping bivalents as single bivalents. In our data set, average proportions of bivalents per cell isolated by the algorithm ranged from `r min(algorithm.stat.category$mean_pass.rate)` (*molossinus^MSM^* male) to `r max(algorithm.stat.category$mean_pass.rate)` (*musculus^KAZ^* female). From the total set of pachytene cell images, `r formatC(  length(unique(Curated_BivData$Obj.ID ) ) , format="d", big.mark=",")` bivalent objects were isolated by the algorithm. Following a manual curation, `r formatC( length(unique(cleanCurated_BivData %>% filter(hand.foci.count != ""))$Obj.ID ), format="d", big.mark=",")` single-bivalent observations remained. The accuracy of the algorithm is high compared to hand measures after this curation step [@peterson2019]. The curated single bivalent data supplements our cell-wide MLH1 count data with MLH1 foci counts for single bivalents. Proportions of bivalents with the same number of MLH1 foci were compared across strains using a chi-square test.

<new 3/n>
To account for confounding effects of sex chromosomes from pooled samples of bivalents, we also considered a reduced data set including only bivalents with SC lengths below the 2nd quartile in cells with at least 17 of 20 single bivalent measures. This “short bivalent” data set included the four or five shortest bivalents and excluded the X bivalent in oocytes. A total of `r sum(Short.biv_mouse_table$n_obs )` short bivalents were isolated from `r sum(  (Short.biv_mouse_table  %>% filter(sex=="female") )$Ncells)` oocytes and `r sum(  (Short.biv_mouse_table  %>% filter(sex=="male") )$Ncells)` spermatocytes. Although this smaller data set has decreased power, it offers a more comparable set of single bivalents to compare between the sexes. A “long bivalent” data set was formed from those bivalents above the 4th quartile in SC lengths per cell. A total of `r sum(Long.biv_mouse_table$n_obs)` long bivalents were isolated from `r sum(  (Long.biv_mouse_table  %>% filter(sex=="female") )$Ncells)` oocytes and `r sum(  (Long.biv_mouse_table  %>% filter(sex=="male") )$Ncells)` spermatocytes.

<new 4/n>
To examine crossover interference, the distance (in SC units) between MLH1 foci (inter-focal distance; IFD~raw~) was measured for those single bivalents containing two MLH1 foci. A normalized measure of interference (IFD~norm~) was computed by dividing IFD~raw~ by SC length on a per-bivalent basis.

<new 5/n>
We used a series of statistical models to interpret patterns of variation in the recombination traits we measured (Table 2). We used mouse average as the dependent variable in all analyses. We first constructed a linear mixed model (M1) using lmer() from the lmer4 package [@lme4] in R (v3.5.2) [@Rstudio]. In this model, strain was coded as a random effect, with significance evaluated using a likelihood ratio test (using exactRLRT() from RLRsim [@RLRsim]). Subspecies, sex, and their interaction were coded as fixed effects, with significance evaluated using a chi-square test comparing the full and reduced models (drop1() and anova()) [@lme4]. As each observation was at the level of a single mouse (mouse average), it was uniquely coded within the dataset, nesting was implicit, each strain only occurs within one strain, and was not explicitly coded in our linear and mixed models. We used the subspecies effect to quantify divergence between subspecies and the (random) strain effect to quantify variation within subspecies in a sex-specific manner. In separate analyses using model M1, we considered mouse averages as dependent variables for each of the following traits: MLH1 count per cell, total SC length per cell, single bivalent SC length per cell, IFD~raw~, IFD~norm~, and average MLH1 position (for single-focus bivalents). Four additional lnear models containing only fixed effects (M2-M5) (Table 2) were used to further investigate results obtained from model M1.


# RESULTS


```{r table.for.plot, echo=FALSE, warning=FALSE, message=FALSE}
#why do I add in a different table?
#this move this to the setup file
#MLH1_data  %>% filter(quality < 5)  -- low quality cells removed

#pooled
strain.table.HQ_Fig.og <- ddply( MLH1_data  %>% filter(quality < 5), c("subsp","strain", "sex"), summarise,
                        Nmice =length(unique(mouse)),
                        Ncells  =length(nMLH1.foci),
                        
                        mean_co =as.numeric(format(round(mean(nMLH1.foci), 4 ), nsmall=3) ),
                        mean_co_long = mean(nMLH1.foci),
                        cV = cv(nMLH1.foci, na.rm = TRUE),
                        var = as.numeric(format(round(   var(nMLH1.foci, na.rm = TRUE),3), nsmall=3)), 
                        sd   = round(sd(nMLH1.foci, na.rm = TRUE), 4),
                       #Standard error (mice vs cells)
                        se  = round(sd / sqrt(Ncells), 4)
                       
                       #range_mouse_means = range()
                   #consider adding crhm proportions?
    )

#
strain.table.HQ_Fig.og <- add_subsp(strain.table.HQ_Fig.og)
strain.table.HQ_Fig.og <- add_species(strain.table.HQ_Fig.og)

strain.table.HQ_Fig <-  strain.table.HQ_Fig.og %>% filter(strain != "F1" & strain != "PERC" &
                       strain != "TOM" &  strain != "AST" & strain != "CZECH" & strain != "CAROLI" & strain != "HMI")

#adjust order of plot here
strain.table.HQ_Fig$strain <- factor(strain.table.HQ_Fig$strain,levels =c(
     "MSM", "MOLF",   
  "PWD", "SKIVE",  "KAZ",
  "WSB","G","LEW","PERC",
                     "CAST",
                         "SPRET","SPIC" ), order=T )

```


```{r MLH1.summ.stats, echo=FALSE}

#strain.table.HQ_Fig.og  all strains
#strain.table.HQ_Fig  extra strains removed

main.table <- strain.table.HQ_Fig.og  %>% filter(strain != "F1")

main.table <- add_subsp(main.table)
main.table <- add_species(main.table)

```


```{r MAIN.TABLE1, echo=FALSE, eval=FALSE}

#write the full table to a supplemental table
#show the head (or shortened)
#kable(table(AP_mouse_table.HQ$category), col.names = c("Category", "mice"), label = "Number of mice by category" )
#kable(strain.table.HQ_Fig, "latex")

df.strain.table <- strain.table.HQ_Fig
#kable(df.strain.table)
#pander(df.strain.table)

#kable(strain.table.HQ_Fig.og, col.names = c("Subspecies", "Strain","Sex", "Number of Mice", #"Number of Cells", "Mean MLH1 count per cell", "cV", "Var", "sd", "se"), caption = "Strain #Level Summary Statistics" )  %>%
#  kable_styling() %>%
#  scroll_box(width = "800px", height = "200px")

#kable(table(MLH1_data$category), col.names = c("Category","Cells"), label = "Number of Cells by category" )

#write excel
#write.xlsx2(strain.table.HQ_Fig.og, "second.MLH1.table.xlsx", sheetName = "Sheet1", 
#  col.names = TRUE, row.names = FALSE, append = FALSE)

```


```{r corMLH1.FM, echo=FALSE}

#needs to be strain
sperman.corr.FM <- cor( ( strain.table.HQ_Fig %>% filter(sex == "female") %>% filter(species == "M.musculus" ) )$mean_co, ( strain.table.HQ_Fig %>% filter(sex == "male") %>% filter(species == "M.musculus" ) )$mean_co, 
                        method = c("spearman") )

sperman.corr.FM.pval <- cor.test( ( strain.table.HQ_Fig %>% filter(sex == "female") %>% filter(species == "M.musculus" ) )$mean_co, ( strain.table.HQ_Fig %>% filter(sex == "male") %>% filter(species == "M.musculus" ) )$mean_co, 
                        method = c("spearman") )

```



## Genome-wide recombination rate evolves differently in females and males

<new 1/n>

We used counts of MLH1 foci per cell to estimate genome-wide recombination rates in `r length(unique(  (main.table  %>% filter(species == "M.musculus") )$strain) )` wild-derived inbred strains sampled from three subspecies of house mice (*M. m. domesticus*, _M. m. musculus_ and _M. m. molossinus_ ) and three additional species of Mus (*M. spretus*, _M. spicilegus_ , and *M. caroli*). Mean MLH1 focus counts for `r sum(main.table$Nmice)` mice were quantified from an average of `r mean( (Mouse.Table_HQ %>% filter(sex == "male") )$Ncells, na.rm = T)` spermatocytes per male (for a total of `r formatC( sum( (Mouse.Table_HQ %>% filter(sex == "male") )$Ncells, na.rm = T), format="d", big.mark=",")` spermatocytes) and `r mean( (Mouse.Table_HQ %>% filter(sex == "female") )$Ncells, na.rm = T)` oocytes per female (for a total of `r formatC(sum( (main.table %>% filter(species == "M.musculus") %>% filter(sex == "female"))$Ncells), format="d", big.mark=",")` oocytes).

<new 2/n>

Graphical comparisons reveal sex-specific dynamics to the evolution of genome-wide recombination rate (Figure 1A). First, MLH1 focus counts differ between females and males in most strains. Second, the difference in counts between the sexes varies among strains. Although most strains show more MLH1 foci in females, two strains (*musculus^PWD^* and _molossinus^MSM^_) exhibit higher counts in males. In females, numbers of MLH1 foci are evenly distributed around the sex-wide mean of approximately 25 (Figure 1B). In stark contrast, males largely separate into two groups of strains with high numbers (near 30) and low numbers (near 23) of foci (Figure 1C). Strain mean MLH1 focus counts from females and males are uncorrelated (Spearman’s $\rho$ = 0.08 - `r sperman.corr.FM`; p = 0.84 -- `r sperman.corr.FM.pval$p.value`) across the set of strains.


<new 3/n>

To further partition variation in recombination rate, we fit a series of linear models to mean MLH1 focus counts from `r length( DF.HetC.MixedModel.HQ$mean_co)` house mice (Table 2; detailed results available in Supplemental Table X). Strain, sex, subspecies, and sex\*subspecies each affect MLH1 focus count in a linear mixed model (M1; strain (random effect): p < 10-4 -- `r 1/10000`; sex: p = 6.2x10-4 -- `r MLH1.MMlmer_results$sex.results`; subspecies: p = 1.1x10-3 -- `r MLH1.MMlmer_results$sub.sp`; subspecies\*sex: p = 2x10-4 -- `r MLH1.MMlmer_results$interaction.pval`). 

The effect of subspecies is no longer significant in a model treating all factors as fixed effects **(M2; p = 0.09 -- )**, highlighting strain and sex as salient variables. Two strains exhibit particularly strong effects on MLH1 focus count (M3; *domesticus^G^* p = `r summ.MLH1.M3.strain.fixed$coefficients[,4][3]` ; *domesticus^LEW^* p = `r summ.MLH1.M3.strain.fixed$coefficients[,4][4]`), with sex-strain interactions involving three strains (M3; *domesticus^G^* p = `r summ.MLH1.M3.strain.fixed$coefficients[,4][10]`; *molossinus^MSM^* p = `r summ.MLH1.M3.strain.fixed$coefficients[,4][13]` ; *musculus^PWD^* p = `r summ.MLH1.M3.strain.fixed$coefficients[,4][12]`). 

In separate analyses of males (M4; n = 80 - `r length(male_mouse.avs_for.analysis$mouse)`), three strains disproportionately shape MLH1 focus count (as observed in Figure 1C): *musculus^PWD^* (p = `r sum_MLH1.male.glm.M4$coefficients[,4][7]` ; effect = `r sum_MLH1.male.glm.M4$coefficients[7]` ), *molossinus^MSM^* (p = `r sum_MLH1.male.glm.M4$coefficients[,4][8]`; effect = `r sum_MLH1.male.glm.M4$coefficients[8]`), and *musculus^SKIVE^* (p = `r sum_MLH1.male.glm.M4$coefficients[,4][9]`; effect = `r sum_MLH1.male.glm.M4$coefficients[9]` ). These three strains point to substantial evolution in the genome-wide recombination rate in spermatocytes; we subsequently refer to them as “high-recombination” strains. 

In females (M4; n= 76 - `r length(female_mouse.avs_for.analysis$mouse)`), four strains affect MLH1 focus count: *domesticus^G^* (p = `r sum_MLH1.female.spc.glm.M5$coefficients[,4][2]`; effect = `r sum_MLH1.female.spc.glm.M5$coefficients[2]`), *molossinus^MSM^* (p = `r sum_MLH1.female.spc.glm.M5$coefficients[,4][5]`; effect = `r sum_MLH1.female.spc.glm.M5$coefficients[5]`), *domesticus^LEW^* (p = `r sum_MLH1.female.spc.glm.M5$coefficients[,4][3]` ; effect = `r sum_MLH1.female.spc.glm.M5$coefficients[3]`), and *musculus^PWD^* (p = `r sum_MLH1.female.spc.glm.M5$coefficients[,4][4]`; effect = `r sum_MLH1.female.spc.glm.M5$coefficients[4]`). Strain effect sizes in females are modest in magnitude compared to those in males.  


Together, these results demonstrate that the genome-wide recombination rate evolves in a highly sex-specific manner.

## Synaptonemal complexes are longer in females

<new 1/n>
The variation in sex differences in recombination we discovered provided an opportunity to determine whether sex differences in chromatin compaction, as measured by the length of the synaptonemal complex (SC), are reversed when heterochiasmy is reversed. In all strains except _musculus^SKIVE^_, females have longer SCs than males, whether SC length was estimated as the total length across bivalents or as the length of short bivalents (t-tests; all p < 0.05, except short bivalents in _musculus^SKIVE^_, p = 0.11 --  `r Q1_short.SC_t.test.SKIVE$p.value``). 

Among short bivalents (to which the female X bivalent does not contribute), female to male ratios of mouse mean SC length range from 1.26 - `r Q1SC.mus.av.short.biv_DIFF_PWD`  (*musculus^PWD^*) to 1.49 - `r Q1SC.mus.av.short.biv_DIFF_WSB` (*domesticus^WSB^*) across strains. That females have longer SCs is further supported by models that include covariates, which identify sex as the most consistently significant effect for total SC length (M1: p = 7.16^{-20} -- `r Q1.lmer_totSC_M1_results$sex.results` ; M2: p = 5.33^{-4} -- `r sum.Q1.glm_totSC_M2$coefficients[,c(1,4)][9,2]`; M3: p =0.05 -- `r sum.Q1.glm_totSC_M3$coefficients[,c(1,4)][8,2]` ) and short bivalent length (M1: p = 1.27^{-11 -- `r Q1.lmer_short_M1_results$sex.results`}; M2: p = 1.98^{-7} -- `r summ.Q1.glm_short.SC_M2$coefficients[,c(1,4)][8,2] / 10000`; M3: p = 1.98^{-7} -- `r summ.Q1.glm_short.SC_M3$coefficients[,c(1,4)][8,2] / 10000`). 

The existence of some subspecies and strain effects (M1 subspecies\*sex: p = 0.03 -- `r Q1.lmer_short_M1_results$interaction.pval`; M2: strain-*musculus^SKIVE^* : p=0.02 -- `r summ.Q1.glm_short.SC_M2$coefficients[7,c(1,4)][2]` ; _musculus^SKIVE^_\*male: p = 0.08 -- `r summ.Q1.glm_short.SC_M2$coefficients[13,c(1,4)][2]`) further indicates that SC length has evolved among strains and among subspecies.

In summary, two approaches for measuring SC length demonstrate that females have longer SCs (chromosome axes), even in strains in which males recombine more. This pattern implies that in high-recombination strains, spermatocytes have less space than oocytes in which to position additional crossovers.

**double check that these are the only sig factors for all the models**

--------

## Females and males differ in crossover positions and crossover interference

<new 1/n>
We used normalized positions of MLH1 foci along bivalents with a single focus to compare crossover location while controlling for differences in SC length. In all strains, MLH1 foci tend to be closer to the telomere in males (mean normalized position in males: 0.68--`r mean( (mouse.level.BivData_table %>% filter(sex == "male") %>% filter(species == "M.musculus") )$norm.first.foci.pos, na.rm = TRUE )`; mean normalized position in females: 0.56 --  `r mean( (mouse.level.BivData_table %>% filter(sex == "female") %>% filter(species == "M.musculus") )$norm.first.foci.pos, na.rm = TRUE )`; t-test; p = 3.84^{-23} -- `r Q1_1CO.pos_t.test.FULL$p.value`). Sex is also the strongest determinant of MLH1 focus position in the models we tested (M1: p = 7.79^{-26 --  `r lmer.Q1.nrm1CO_M1_results$sex.results`}; M2: p = 5.39^{-8} --  `r summry.Q1.glm_F1pos_M2$coefficients[,4][8]`; M3: p = 5.39^{-8} --  `r summry.Q1.glm_1CO.pos_M3$coefficients[,4][8]`).

<new 2/n>
Males have longer normalized mean inter-focal distances (IFD~norm~) than females in seven out of eight strains (t-tests; p < 0.05), with only _musculus^KAZ^_ showing no difference (p = 0.33 -- `r Q1.IFD.PER_t.test_KAZ$p.value`). Examination of IFD~norm~ distributions indicates that females are centered at approximately 50% and show a slight enrichment of low (<25%) values, whereas males are enriched for higher values. Models treating IFDnorm as the dependent variable support the inference of stronger interference in males, with sex being the most significant variable (M1: p = 3.11^{-12} -- `r Q1.IFD.PER_M1_lmer.results$sex.results` ; M2: p = 0.01  --  `r summry.glm.Q1_IFD.PER.M2$coefficients[,4][4]`; M3: p = 0.01  --  `r summry.glm.Q1_IFD.PER.M3$coefficients[,4][2]`). In contrast, there is no clear signal of sex differences in raw mean inter-focal distances (IFD~raw~) across the full set of strains, whether they are considered separately or together. Visualization of normalized MLH1 foci positions on bivalents with two crossovers (Figure 4C) further suggests that interference distances vary more in females than in males, and that males display a stronger telomeric bias in the placement of the distal crossover. 

In summary, controlling for differences in SC length (chromatin compaction) indicates that interference is stronger in males, whereas interference on the physical scale is similar in the sexes.

--------


## Evolution of genome-wide recombination rate is dispersed across bivalents, associated with double-strand break number, and connected to crossover interference


```{r algorithm_performance_stats, echo=FALSE, eval=FALSE}

#number of mice, number of cells
#mean number of isolate bivs per cell,
#use these two metrics
#cell.level.BivData_table$boxs.IDd 
#cell.level.BivData_table$nboxes_passed

#calq the mean rate by category

cell.level.BivData_table$pass.rate <- cell.level.BivData_table$nboxes_passed / cell.level.BivData_table$boxs.IDd 

algorithm.stat.mouse <- ddply(cell.level.BivData_table, c("mouse"), summarise,
                        n_obs = length( unique(fileName) ),
                         mean_pass.rate = mean(pass.rate)
                         )

cell.level.BivData_table <- add_sex(cell.level.BivData_table)

algorithm.stat.category <- ddply(cell.level.BivData_table, c("sex", "strain"), summarise,
                        n_obs = length(unique(fileName)),
                        mean_pass.rate = mean(pass.rate)
                         )

#showing the table..
#data.frame(table( droplevels( Curated_BivData  %>%  filter(hand.foci.count  == 0 | hand.foci.count  == 1 |                                       hand.foci.count  == 2 | hand.foci.count  == 3 | hand.foci.count  == 4 ) )$category ) )
```


<new 1/n>
We used the contrast between males from high-recombination strains and males from low-recombination strains to identify features of the recombination landscape associated with evolutionary transitions in the genome-wide recombination rate. We considered proportions of bivalents with different numbers of crossovers, double-strand break number, SC length, and crossover positioning.

<new 2/n>
Ninety-six percent of single bivalents in our pooled dataset (n = 9,576 -- `r formatC( sum( Chrm.prop_strain_table$n0CO, Chrm.prop_strain_table$n1CO, Chrm.prop_strain_table$n2CO, Chrm.prop_strain_table$n3CO, Chrm.prop_strain_table$n4CO ), format="d", big.mark=",")` ) have either one or two MLH foci (Figure 3). The proportions of single-focus (1CO) bivalents vs. double-focus (2CO) bivalents distinguish high-recombination strains from low-recombination strains (Figure 3). High-recombination strains are enriched for 2CO bivalents at the expense of 1CO bivalents: proportions of 2CO bivalents are 0.33 -- `r chrm.prop.male.high1$n2CO[chrm.prop.male.high1$strain == "SKIVE"] / chrm.prop.male.high1$nbiv[chrm.prop.male.high1$strain == "SKIVE"]` in _musculus^SKIVE^_, 0.44 -- `r chrm.prop.male.high1$n2CO[chrm.prop.male.high1$strain == "PWD"] / chrm.prop.male.high1$nbiv[chrm.prop.male.high1$strain == "PWD"]` in _musculus^PWD^_ , and 0.51 -- `r chrm.prop.male.high1$n2CO[chrm.prop.male.high1$strain == "MSM"] / chrm.prop.male.high1$nbiv[chrm.prop.male.high1$strain == "MSM"]` in _molossinus^MSM^_ (Supplemental Figure 3). Following patterns in the genome-wide recombination rate, male _musculus^PWD^_ and male _molossinus^MSM^_ have 2CO proportions that are more similar to each other than to strains from their own subspecies (chi-square tests; _musculus^PWD^_ vs. *molossinus^MSM^*: p = 0.37 -- `r PWD.MSM.2CO.prop.test$p.value` ; _musculus^PWD^_ vs. _musculus^KAZ^_: p = 1.23^{-31} -- `r PWD.KAZ.2CO.prop.test$p.value`; _molossinus^MSM^_ vs. _molossinus^MOLF^_: p = 2.65^{-6} -- `r MSM.MOLF.2CO.prop.test$p.value` ). These results demonstrate that evolution of the genome-wide recombination rate reflects changes in crossover number across multiple bivalents.

<new 3/n>
To begin to localize evolution of genome-wide recombination rate to steps of the recombination pathway, we counted DMC1 foci in prophase spermatocytes as markers for double-strand breaks (DSBs). DMC1 foci were counted in a total of 76 -- `r sum( (DMC1table.main$ncells.x) )` early zygotene and 75 -- `r sum( (DMC1table.main$ncells.y) )` late zygotene spermatocytes from two high-recombination strains (*musculus^PWD^* and *molossinus^MSM^*) and three low-recombination strains (*musculus^KAZ^*, _domesticus^WSB^_, and _domesticus^G^_). High-recombination strains have significantly more DMC1 foci than low-recombination strains in early zygotene cells (t-test; p < 10^{-6} -- `r 1/1000000` ). In contrast, the two strain groups do not differ in DMC1 foci in late zygotene cells (t-test; p = 0.66 -- `r ttest.HighLow.Z.pval` ). Since DSBs are repaired as either COs or non-crossovers (NCOs), the ratio of MLH1 foci to DMC1 foci can be used to estimate the proportion of DSBs designated as COs. High-recombination and low-recombination strains do not differ in the MLH1/DMC1 ratio, whether DMC1 foci were counted in early zygotene cells or late zygotene cells (t-test; p > 0.05). These results raise the possibility that the evolution of genome-wide recombination rate is primarily determined by processes that precede the CO/NCO decision, at least in house mice.

<new 4/n>
Total SC length only partially differentiates high-recombination strains from low-recombination strains (Figure 5). Whereas high-recombination strains as a group have significantly greater total SC length than low-recombination strains (t-test; p = 0.01 -- `r Q2tot.sc_t.test_High.Low$p.value` ), separate tests within subspecies show that the two strain categories differ within *M. m. molossinus* (p = 0.02 -- `r Q2tot.sc_t.test_High.Low_Mol$p.value` ) but not within *M. m. musculus* (p = 0.4 -- `r Q2tot.sc_t.test_High.Low_Musc$p.value` ). Additionally, mouse means for the reduced (short and long) bivalent datasets do not differ between high-recombination and low-recombination strains (t-test; short: p = 0.88 -- `r Q2short.biv_t.test_High.Low$p.value`  ; long: p = 0.19 --  `r Q2long.biv_t.test_High.Low$p.value` ). 

Write results for M4 and M5 here
total SC M4 results

Musc subspecies very ***, Mol *** (both ++)
G, LEW, 
MSM
SKIVE, KAZ (- -)

M5 (almost all strains are significant)
G, LEW, PWD, MSM
MOLF, 
SKIVE, KAZ, TOM, AST, CZECH


In a model with total SC length as the dependent variable (*M2?*), two subspecies effects are significant (*M. m. musculus* p = 2.33^{-6} -- `r Q2tot.sc_t.test_High.Low_Musc$p.value`, *M. m. molossinus*, p = 21^{-6} -- `r Q2tot.sc_t.test_High.Low_Mol$p.value`). In models with SC lengths of short and long bivalents as dependent variables, several subspecies and strain effects reach significance (p < 0.05), but they are not consistent across models. Collectively, these results reveal that evolution of SC length is not strongly associated with evolution of genome-wide recombination rate in house mice.

<new 5/n>
In summary, evolution of the genome-wide recombination rate in males is connected to double-strand break number and crossover interference, but not to SC length and crossover position (on single-crossover bivalents).

--------------------------


# DISCUSSION

<new 1/n>
By comparing recombination rates in females and males from the same diverse set of genetic backgrounds, we isolated sex as a primary factor in the evolution of this fundamental meiotic trait. Recombination rate differences are more pronounced in males than females. Because inter-strain divergence times are identical for the two sexes, this observation demonstrates that the genome-wide recombination rate evolves faster in males, at least in house mice. More generally, recombination rate divergence is decoupled in females and males. These disparities are remarkable given that recombination rates for the two sexes were measured in identical genomic backgrounds (other than the presence/absence of the Y chromosome). Our results provide the strongest evidence yet that the genome-wide recombination rate follows distinct evolutionary trajectories in males and females.

<new 2/n>
At the genetic level, the sex-specific evolution we documented indicates that some mutations responsible for divergence in recombination rate have dissimilar phenotypic effects in females and males. A subset of the genetic variants associated with genome-wide recombination rate within populations of humans [@Kong2004;, @Kong2008; @Kong2014; @halldorsson2019], Soay sheep [@johnston2016_soay], and cattle [@ma2015_cattle; @Shen2018_cattle] appear to show sex-specific properties, including opposite effects in females and males. Furthermore, inter-sexual correlations for recombination rate are weak in humans [@fledel2011] and Soay sheep [@johnston2016_soay]. Crosses between the strains we surveyed could be used to identify and characterize the genetic variants responsible for recombination rate evolution in house mice [@dumont2011; @Wang2017island; @wang2019_SC]. These variants could differentially affect females and males at any step in the recombination pathway. Although our DMC1 profiling was limited to males from a small number of strains (for practical reasons), our findings suggest that mutations that determine the number of double-strand breaks contribute to sex-specific evolution in the recombination rate. A study of two classical inbred strains and one wild-derived inbred strain of house mice also found a positive association between crossover number and double-strand break number in males [@baier2014].

<new 3/n>
Another implication of our results is that the connection between recombination rate and fitness differs between males and females. Little is known about whether and how natural selection shapes recombination rate in nature [@DapperPayseur2017; @Ritz2017].  @samuk2020 recently used a quantitative genetic test to conclude that an 8% difference in genome-wide recombination rate between females from two populations of *Drosophila pseudoobscura* was caused by natural selection. Applying similar strategies to species in which both sexes recombine, including house mice, would be a logical next step to understanding the sex-specific evolution of recombination rate. 

<new 4/n>
Population genetic models have been built to explain sexual dimorphism in the number and placement of crossovers, which is a common phenomenon [@brandvain2012scrambling; @sardell_sex_2020]. Modifier models predicted that lower recombination rates in males will result from haploid selection [@lenormand2003] or sexually antagonistic selection on coding and cis-regulatory regions of genes [@sardell_sex_2020]. Another modifier model showed that meiotic drive could stimulate female-specific evolution of the recombination rate [@brandvain2012scrambling]. Although these models fit the conserved pattern of sex differences in crossover positions, they do not readily explain our observations of sex-specific evolution in the genome-wide recombination rate. In particular, the alternation across strains in which sex has more crossovers is unexpected.

<new 5/n>
We propose an alternative interpretation of our findings based on the cell biology of gametogenesis. During meiosis, achieving a stable chromosome structure requires the attachment of kinetochores to opposite poles of the cell and at least one crossover to create tension across the sister chromosome cohesion distal to chiasmata [@LaneKauppi2019; @vanVeen2003]. The spindle assembly checkpoint (SAC) prevents aneuploidy by ensuring that all bivalents are correctly attached to the microtubule spindle (“bi-oriented”) before starting the metaphase-to-anaphase transition via the release of the sister cohesion holding homologs together [@LaneKauppi2019]. Hence, selection seems likely to favor mutations that optimize the process of bi-orientation and chromosome separation, thereby prohibiting the SAC from delaying the cell cycle or triggering apoptosis. Multiple lines of evidence indicate that the SAC is more effective in spermatogenesis than in oogenesis [@LaneKauppi2019], perhaps due to the presence of the centrosome spindle [@So2019] and larger cell volume [@kyogoku2017] in oocytes. The higher stringency of the SAC during spermatogenesis suggests that selection will be better at removing mutations that interfere with bi-orientation in males than in females. Therefore, faster male evolution of the genome-wide recombination rate could be driven by the more stringent SAC acting on chromosome structures at the metaphase I alignment.

<new 5/n>
Our SAC model is consistent with other features of our data. We showed that widespread sex differences in broad-scale crossover positioning [@sardell_sex_2020] apply across house mice, even in lineages where the direction of heterochiasmy is reversed. The number and placement of crossovers affects the area of sister chromosome cohesion distal to crossovers which needs to be released for the first reductional chromosome segregation [@vanVeen2003; @LaneKauppi2019; @subramanian2014; @dumontDesai2012]. Faster spermatogenesis may select for synchronization of the separation across all homologs within the cell (**Kudo** ), whereas in oogenesis, the slower cell cycle and multiple arrest stages may require chromosome structures with greater stability on the MI spindle, especially for those organisms that undergo dictyate arrest [@Lee2019].

<new 6/n>
We propose that the SAC model also can explain the correlated evolution of stronger crossover interference and higher genome-wide recombination rate in male house mice. Our results show that crossovers are spaced further apart in strains enriched for double-crossover bivalents when SC length is taken into account and chromosome size effects are minimized. Assuming chromatin compaction between (prophase) pachytene and metaphase is uniform along bivalents, this increased spacing is expected to expand the area for sister cohesion to connect homologs and may improve the fidelity of chromosomal segregation. While the SAC model postulates direct fitness effects of interference, a modifier model predicted that indirect selection on recombination rate – via its modulation of offspring genotypes – can strengthen interference as well [@goldstein1993].

<new 7/n>
Regardless of the underlying mechanism, our results provide a rare demonstration that crossover interference can diverge over short evolutionary timescales. The notion that stronger interference can co-evolve with higher genome-wide recombination rate is supported by differences between breeds of cattle (@ma2015_cattle) and differences between populations of white-footed mice [@peterson2019]. In contrast, mammalian species with stronger interference tend to exhibit lower genome-wide recombination rates (@segura2013, @ottoPaysuer2019). Collectively, these patterns suggest that inferences about the evolutionary dynamics of interference depend on the timescale under consideration.

<new 8/n>
Our findings further reveal that evolution of the genome-wide recombination rate does not require major changes in the degree of chromatin compaction. Female house mice consistently show longer SCs, even in strains with more recombination in males. Studies in mice [@lynn2002, @petkov2007] and humans [@gruhn2013, @tease2004] suggest that chromosomal axes are longer (and DNA loops are shorter) in females than males. Some authors have suggested that conserved sex differences in crossover positioning (more uniform placement in females) and interference strength (stronger interference in males) could be due to looser chromatin packing of the meiotic chromosome structure in females [@haenel2018; @petkov2007]. A cellular model designed to explain interference attributes sexual dimorphism in chromatin structure to greater cell volumes and oscillatory movements of telomeres and kinetochores in oocytes [@hulten2011_COM]. More recent work connects the sparser recombination landscape to sex differences in the crossover repair pathway [@wang2017inefficient].

<new 9/n>
Our conclusions are accompanied by several caveats. First, MLH1 foci only identify interfering crossovers [@holloway2008mus81]. Although most crossovers belong to this class **(REF)**, our approach likely underestimated genome-wide recombination rates. Evolution of the number of non-interfering crossovers is a topic worth examining. A second limitation is that our investigation of crossover locations was confined to the relatively low resolution possible with immunofluorescent cytology. Positioning crossovers with higher resolution could reveal additional evolutionary patterns. Finally, the panel of inbred lines we surveyed may not be representative of recombination rate variation within and between subspecies of house mice. We considered most available wild-derived inbred lines, but house mice have a broad geographic distribution. Nevertheless, we expect our primary conclusion that recombination rate evolves in a sex-specific manner to be robust to geographic sampling because differences between females and males exist for the same set of inbred strains.

<new 10/n>
While the causes of sex differences in recombination remain mysterious @lenormand2016, our conclusions have implications for a wide range of recombination research. For biologists uncovering the cellular and molecular determinants of recombination, our results suggest that mechanistic differences between the sexes could vary by genetic background. For researchers charting the evolutionary trajectory of recombination, our findings indicate that sex-specific comparisons are crucial. For theoreticians building evolutionary models of recombination, different fitness regimes and genetic architectures in females and males should be considered. Elevating sex as a primary determinant of recombination would be a promising step toward integrating knowledge of cellular mechanisms with evolutionary patterns to understand recombination rate variation in nature.


--------

# REFERENCES

